lib/web_ui/lib/src/engine/canvaskit/picture.dart

```diff
@@ -101,12 +101,12 @@ class CkPicture extends ManagedSkiaObject<SkPicture> implements ui.Picture {
   @override
   ui.Image toImageSync(int width, int height) {
     assert(debugCheckNotDisposed('Cannot convert picture to image.'));
-    final DomCanvasElement tempCanvas =
-        createDomCanvasElement(width: width, height: height);
-    final SkSurface skSurface = canvasKit.MakeWebGLCanvasSurface(tempCanvas);
-    final SkCanvas skCanvas = skSurface.getCanvas();
-    skCanvas.drawPicture(skiaObject);
-    final SkImage skImage = skSurface.makeImageSnapshot();
+    final Surface skSurface = Surface();
```

Comment:
nit: I'd rename the variable, since it's not a Skia surface.

Reply:
Done